### DISCUSSION TITLE
What is the best course for AI learning?

### DISCUSSION THREAD

**Discussion by solyasa** [community/community#30122] [Other Feature Feedback, Questions, & Ideas] [answered]
As you know, there are many course for AI learning.
As I am beginner of AI, I want to know the best AI course.

I like AI tech and I have passion about that.
Best regards

**Comment by levintech** [comment, marked as answer]
You can find useful courses on [Coursera](https://www.coursera.org/).
I think this course [AI for Everyone](https://www.coursera.org/learn/ai-for-everyone) would be good start.
Hope you success.

**Reply by solyasa** [reply]
Thank you
